### ISSUE TITLE
Graphical installer ISOs are not accessible

### ISSUE THREAD

**Issue by emilazy** [NixOS/nixpkgs#339070]
### Describe the bug
Users of screen readers cannot reliably install NixOS with the graphical installer ISOs.

### Steps To Reproduce
Boot the GNOME or Plasma installation ISOs and try to start a screen reader (e.g., with <kbd>Alt</kbd>+<kbd>F2</kbd> `orca` <kbd>Enter</kbd>). It may fail to start (because of not being present, or a strange timeout error), but even if it does start, it doesn’t read out our Qt‐based Calamares installer.

### Expected behavior
A screen reader reliably starts and can be used to interact with the installer.

### Notify maintainers
cc @K900 @ndarilek

**Comment by emilazy** [comment]
See https://github.com/NixOS/nixpkgs/pull/339069 for including Orca in the Plasma ISOs (but it may not fix interacting with the installer and doesn’t handle whatever weird thing is going on with <kbd>Alt</kbd>+<kbd>F2</kbd> in GNOME).

**Comment by emilazy** [comment]
Apparently <kbd>Meta</kbd>+<kbd>Alt</kbd>+<kbd>S</kbd> is the shortcut to start the screen reader on both GNOME and Plasma. It works for me on the GNOME ISO, but takes an unusual‐seeming amount of time to start in my VM (~10 seconds?). It still does not read out the installer window, which suggests to me that we may be missing some Qt accessibility glue in the GNOME installer ISO.

**Comment by emilazy** [comment]
Another issue here is that we don’t include Orca by default on installed systems. We should fix that too, or the installer will only be usable to create an unusable machine.
